pyproject.toml

```diff
@@ -13,6 +13,7 @@ backoff = "^2.2.1"
 milagro-bls-binding = "==1.9.0"
 py-ecc = "^6.0.0"
 ipfshttpclient = "^0.8.0a2"
+faker = "==15.3.4"
```

Comment:
This will cause a `faker` installation in prod that we don't need. It would be great to install it only for the dev environment.